### PR TITLE
[php/ja] Translated PHP tutorial to japanese

### PR DESCRIPTION
**Pull request by kazu9su**
Changes
-------------------
* Create `ja-jp/php.html.markdown` file

### CODE REVIEW

File: ja-jp/php.html.markdown [hunk 1]
```diff
@@ -0,0 +1,776 @@
+---
+language: PHP
+contributors:
+    - ["Malcolm Fell", "http://emarref.net/"]
+    - ["Trismegiste", "https://github.com/Trismegiste"]
+translators:
+    - ["Kazushige Tominaga", "https://github.com/kazu9su"]
+filename: learnphp.php
```
Comment (vendethiel) [inline ja-jp/php.html.markdown]: please add `-jp` suffix, and a newline with `lang: ja-jp`

**Comment by kazu9su** [comment]
@vendethiel 
Thank you for your note.
I fixed.

**Comment by vendethiel** [comment]
Great! thanks, and sorry for the delay

**Comment by vendethiel** [comment]
:star2:

**Comment by kazu9su** [comment]
Thank you! :)

**Pull request merged**